src/components/Contact/Contact.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { StyleSheet, Text, View, ImageBackground } from 'react-native';
+
+
+// Components
+import Header from './Header';
+
+export default class Home extends React.Component {
+  render() {
+    return (
+        <ImageBackground source={require('../../img/img-bg-dark.jpg')} style={styles.imageBackground}>
+          <Header />
+
+
```

Comment:
Contact/Contact.js: Code is clean and well-organized for future additions/amendments.